release/tablelist.js

```diff
@@ -626,24 +618,38 @@ angular
 
       Keychain.prototype.setAuthToken = function(token) {
         _authToken = token || null;
-        return cookie.set(AUTH_KEY, token);
+
+        if (_authToken) {
+          return cookie.set(AUTH_KEY, token);
+
+        } else {
+          return cookie.remove(AUTH_KEY);
+        }
```

Comment:
Note: The auth service will sometimes use ``setAuthToken(null)`` to clear an existing auth token. ``Since cookieService.set(key, value)`` can't take value as ``null``, had to make use of the ``.remove``